### PR TITLE
Add more Remote tests

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: test/e2e/inspect_test.go [hunk 1]
```diff
@@ -132,28 +132,27 @@ var _ = Describe("Podman inspect", func() {
...
 		Expect(inspectSrc.ExitCode()).To(Equal(0))
 		Expect(inspectSrc.OutputToString()).To(Equal("/tmp"))
 
-		inspectDestination := podmanTest.Podman([]string{"inspect", "-l", "--format", "{{(index .Mounts 0).Destination}}"})
+		inspectDestination := podmanTest.Podman([]string{"inspect", "test", "--format", "{{(index .Mounts 0).Destination}}"})
 		inspectDestination.WaitWithDefaultTimeout()
 		Expect(inspectDestination.ExitCode()).To(Equal(0))
 		Expect(inspectDestination.OutputToString()).To(Equal("/test1"))
 
-		inspectDst := podmanTest.Podman([]string{"inspect", "-l", "--format", "{{(index .Mounts 0).Dst}}"})
```
Comment (TomSweeneyRedHat) [inline test/e2e/inspect_test.go:156]: might be good to keep one or two of the --latest tests

**Comment by TomSweeneyRedHat** [comment]
tests aren't too happy

File: test/e2e/inspect_test.go (hunk 1 above)
Comment (rhatdan) [reply test/e2e/inspect_test.go:156]: There are still plenty of tests that use --latest, and maybe we should setup one test for each command to try --latest, but we should not use it when testing other commands.

**Comment by mheon** [comment]
LGTM

**Pull request merged**